src/preferences/dialog/dlgpreferences.cpp

```diff
@@ -371,12 +373,29 @@ void DlgPreferences::onShow() {
         screenSpace = QSize(800, 600);
     }
     else {
-        screenSpace = pScreen->size();
+        screenSpace = pScreen->availableSize();
```

Comment:
`availableSize` returns QSize(width, height)

Did you consider pScreen->[availableGeometry()](https://doc.qt.io/qt-5/qscreen.html#availableGeometry-prop) here?
"The available geometry is the geometry excluding window manager reserved areas such as task bars and system menus." so basically identical 'size' plus position.
`availableGeometry` returns QRect(int x, int y, int width, int height)